### DISCUSSION TITLE
Delta Time :small_red_triangle:

### DISCUSSION THREAD

**Discussion by pulgamecanica** [codam-coding-college/MLX42#128] [General]
Hello, I was wondering if you can recommend me a few places to read about delta time, I've been reading on the recommended results from my engine but I wanted to get a recommendation from you guys. As I understand doing an interpolation with dt should be more accurate than calculating the frames to allow or not to execute the next frame.

Cheers.

**Comment by pulgamecanica** [comment]
I know I can get the fps by doing: `fps = 1000 / mlx->delta_time;`

I want to go deeper about this subject because I am writting a blog about this so I must be very precise! :dagger: 

In the wiki there was not info about delta-time beyond the description. (which is helpful)

Thanks in advance.

**Reply by W2Wizard** [reply]
A straightforward example I found: https://youtu.be/yGhfUcPjXuE?t=131
